### ISSUE TITLE
Empty error file with input processor errors

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#8180]
Issue overview
--------------
While testing another PR, noticed that an idf with an invalid key choice produces an empty error file. Not sure when this started. Sometime after v9.3.0 bugfix release which works fine.

### Details
Some additional details for this issue (if relevant):
 - Platform Windows 10
 - Version of EnergyPlus v9.4.0-https://github.com/NREL/EnergyPlus/commit/e884660c04f1cc6f14ff32f105ee650637069b7f

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added - https://github.com/NREL/EnergyPlusDevSupport/blob/0738e371061a1135ef5fcf95843c52902c2970af/DefectFiles/8000s/8180/8180-1ZoneUncontrolled-develop.idf
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
Works fine on **Ubuntu 18.04** with current develop 922f888aef

```
(py38)julien@8180 (master *%=)$ $ep_build_rel/Products/energyplus -D -d out-ori 8180-1ZoneUncontrolled-develop.idf 
EnergyPlus Starting
EnergyPlus, Version 9.4.0-922f888aef, YMD=2020.08.03 14:31
**FATAL:Errors occurred on processing input file. Preceding condition(s) cause termination.
EnergyPlus Run Time=00hr 00min  0.15sec
Program terminated: EnergyPlus Terminated--Error(s) Detected.

(py38)julien@8180 (master *%=)$ cat out-ori/eplusout.err 
Program Version,EnergyPlus, Version 9.4.0-922f888aef, YMD=2020.08.03 14:31,
   ** Severe  ** <root>[SimulationControl][SimulationControl 1][do_zone_sizing_calculation] - "Maybe" - Failed to match against any enum values.
   **  Fatal  ** Errors occurred on processing input file. Preceding condition(s) cause termination.
   ...Summary of Errors that led to program termination:
   ..... Reference severe error count=1
   ..... Last severe error=<root>[SimulationControl][SimulationControl 1][do_zone_sizing_calculation] - "Maybe" - Failed to match against any enum values.
   ************* Warning:  Node connection errors not checked - most system input has not been read (see previous warning).
   ************* Fatal error -- final processing.  Program exited before simulations began.  See previous error messages.
   ************* EnergyPlus Warmup Error Summary. During Warmup: 0 Warning; 0 Severe Errors.
   ************* EnergyPlus Sizing Error Summary. During Sizing: 0 Warning; 0 Severe Errors.
   ************* EnergyPlus Terminated--Fatal Error Detected. 0 Warning; 1 Severe Errors; Elapsed Time=00hr 00min  0.15sec
```

**Comment by jmarrec** [comment]
Confirmed on Windows.

```
C:\Users\julien\Software\Others\EnergyPlusDevSupport\DefectFiles\8000s\8180>C:\Users\julien\Software\Others\EnergyPlus\build\Products\Debug\energyplus.exe -D -d out-ori 8180-1ZoneUncontrolled-develop.idf
EnergyPlus Starting
EnergyPlus, Version 9.4.0-922f888aef, YMD=2020.08.03 15:19
**FATAL:Errors occurred on processing input file. Preceding condition(s) cause termination.
EnergyPlus Run Time=00hr 00min  6.25sec
Program terminated: EnergyPlus Terminated--Error(s) Detected.

C:\Users\julien\Software\Others\EnergyPlusDevSupport\DefectFiles\8000s\8180>cat out-ori/eplusout.err

C:\Users\julien\Software\Others\EnergyPlusDevSupport\DefectFiles\8000s\8180>
```

**Comment by jmarrec** [comment]
```
Exception thrown at 0x00007FFC1B1CA799 in energyplus.exe: Microsoft C++ exception: EnergyPlus::FatalError at memory location 0x000000AA5C8F9B10.
```

**Comment by jmarrec** [comment]
`Exception thrown at 0x00007FFC1B1CA799 in energyplus.exe: Microsoft C++ exception: EnergyPlus::FatalError at memory location 0x0000008C381698D0. occurred`

![image](https://user-images.githubusercontent.com/5479063/89194941-55dd2600-d5a8-11ea-8594-e66dd7d1018d.png)

**Comment by mjwitte** [comment]
@jmarrec @lefticus git bisect says this commit is the culprit: cbe25e98a90f75b827d5019f38b4ec293e595605 Move err_stream from gio in https://github.com/NREL/EnergyPlus/pull/8162.

**Comment by lefticus** [comment]
OK, you can assign this to me, I'll pick it up hopefully this afternoon along with the other ERL trace assertion error that @JasonGlazer brought up in email

**Comment by lefticus** [comment]
Fixed in #8188

**Comment by lefticus** [comment]
Closing, #8188 has been merged.
